Source/Charts/Data/Implementations/Standard/ChartData.swift

```diff
@@ -91,6 +91,50 @@ open class ChartData: NSObject, ExpressibleByArrayLiteral
         xMin = .greatestFiniteMagnitude
 
         forEach { calcMinMax(dataSet: $0) }
+
+        // left axis
+        let firstLeft = getFirstLeft(dataSets: dataSets)
```

Comment:
Is this coming from another PR? I remember I saw someone saying duplicated calculation, not sure

Reply:
It's coming from new changes in master. Keeping the 4.0.0 branches in sync with master is very difficult.

Reply:
I agree. we should finish 4.0 soon or redirect other PRs to 4.0.